test/seam/connect/client.test.ts

```diff
@@ -159,3 +164,32 @@ test.failing(
     t.true(workspaces.length > 0)
   },
 )
+
+test('SeamHttp: request methods return a SeamApiRequest object', async (t) => {
```

Comment:
Can you move this test to its own file `test/seam/connect/seam-api-request.test.ts`? These tests are for specifically for the `client` option. You can follow simpler pattern to initialize the client as done here: https://github.com/seamapi/javascript-http/blob/main/test/seam/connect/serialization.test.ts#L8